isso/__init__.py

```diff
@@ -281,6 +281,9 @@ def main():
     if conf.get("general", "log-file"):
         handler = logging.FileHandler(conf.get("general", "log-file"))
 
+        if conf.get("general", "log-format"):
+            handler.setFormatter(logging.Formatter(conf.get("general", "log-format")))
```

Comment:
1. What happens when the log format is invalid?
2. Why set the log format only for file logs, instead of for stdout as well?

Reply:
> 1. What happens when the log format is invalid?

IIRC, there is a traceback when logger tries to emit a message.

> 2. Why set the log format only for file logs, instead of for stdout as well?

Because we are usually interested in persisted logs. And because there was only a  `FileHandler` there (and I didn't want to tweak formatting _globally_).